snake.js

```diff
@@ -1,4 +1,4 @@
-const gameTimer = setInterval(draw,125);
+const gameTimer = setInterval(draw, difficulty());
```

Comment:
Not the best decision to put difficulty in this place and design it in this manner.